src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label>
+          Surname:
+          <input 
+            class="first"
+            type="text" 
+            autocomplete="off">
```

Comment:
```suggestion
          <input 
            class="first"
            type="text" 
            autocomplete="off"
          >
```
Закриваюча дужка повинна бути на новому рядку, якщо атрибути переносятся. Давай це скрізь виправимо